Fix more warnings from FindBugs

stweil opened the pull request:
Signed-off-by: Stefan Weil <sw@weilnetz.de>

henning-gerhardt commented on Goobi/src/org/goobi/production/properties/ProcessProperty.java:
```diff
@@ -59,9 +59,12 @@
 	private boolean currentStepDuplicationAllowed = false;
 	
 	public ProcessProperty() {
+		this.container = new Integer(0);
```
IntelliJ IDEA tolds me to remove this unnecessary boxing.

stweil replied:
I think we have more of that kind, so they can be handled in a new commit.

henning-gerhardt replied:
Introducing a known bad usage to change / fix it later is not a good way. We need no more bad usage which must be fixed sometime.

stweil replied:
Sorry, that was a misunderstanding on my side. It's fixed now with an updated PR.

henning-gerhardt commented:
For commit: https://github.com/kitodo/kitodo-production/pull/507/commits/cb89fd05a3d14c0d91047e35b7caf6c930b1e418
You can not remove this private methods which are used by Hibernate:
```
Could not find a getter for datentyp in class de.sub.goobi.beans.Prozesseigenschaft
```
The other removed methods are maybe used too. So please test your changes in a proper test environment.

henning-gerhardt commented on Goobi/src/org/goobi/production/properties/ProcessProperty.java:
```diff
@@ -291,9 +294,6 @@ public boolean isValid() {
 	public void save(Schritt step) {
-		if (this.prozesseigenschaft != null) {
-
-		}
```
save method is not used nor defined in interface. Safe to remove?

stweil replied:
Yes, I assume so. I added a new commit which removes it.

stweil commented:
PR update:
* Removed commit which removed private methods which were reported to be unused but which are obviously used nevertheless.
* Added commit which removes `save` method.

stweil commented:
New PR, addressed @henning-gerhardt comment.
